x-pack/plugins/cases/public/components/configure_cases/index.test.tsx

```diff
@@ -750,6 +750,7 @@ describe('ConfigureCases', () => {
               type: customFieldsConfigurationMock[0].type,
               label: `${customFieldsConfigurationMock[0].label}!!`,
               required: !customFieldsConfigurationMock[0].required,
+              defaultValue: customFieldsConfigurationMock[0].defaultValue,
```

Comment:
What changed and now we have to add it? `customFieldsConfigurationMock[0]` is required.

Reply:
Thanks! Make sense.